### ISSUE TITLE
Some tvshows fail to work with favorites

### ISSUE THREAD

**Issue by dagwieers** [pietje666/plugin.video.vrt.nu#299]
### Describe the bug ###
We match the TV-shows  `programName` with the favorites name, but this is apparently not very standardized as they may be different, e.g. favorites `de-campus-cup` has programName `campus-cup`.

**Comment by mediaminister** [comment]
Indeed, `programName` is unreliable. I noticed this also with the `get_latest_episode` function.
We should always use `programUrl`:
```
//www.vrt.be/vrtnu/a-z/de-campus-cup/
```
The Favorites video-data api uses a slightly shorter `programUrl` format:
```
/vrtnu/a-z/de-campus-cup/
```
And then we have the VRT suggest api that we use to get a-z and category menus. This api uses the ` targetUrl` format:
```
//www.vrt.be/vrtnu/a-z/de-campus-cup.relevant/
```
Luckily, these three `programUrl` variants can be derived from each other. I made a pull request which implements this functionality so the relevant api requests always use `facets[programUrl]` instead of `facets[programName]`: https://github.com/pietje666/plugin.video.vrt.nu/pull/300
